### PR TITLE
Added several useful extensions.

### PR DESCRIPTION
**Pull request by marisks**

### CODE REVIEW

**Comment by marisks** [comment]
The only extension which is .NET only is enumerable for paging. I will move it to Net.Extensions.

**Comment by marisks** [comment]
Moved enumerable extensions to the .NET extensions package.

File: src/Geta.EPi.Extensions/ViewContextExtensions.cs [hunk 1]
```diff
@@ -8,13 +10,23 @@ namespace Geta.EPi.Extensions
...
         {
             return viewContext.HttpContext.IsBlockPreview();
         }
+
+        /// <summary>
+        /// Checks if block is in edit mode.
+        /// </summary>
+        /// <param name="context"></param>
+        /// <returns></returns>
+        public static bool IsBlockInEditMode(this ViewContext context)
```
Comment (valdisiljuconoks) [inline src/Geta.EPi.Extensions/ViewContextExtensions.cs]: does this apply only on blocks?

Comment (marisks) [reply src/Geta.EPi.Extensions/ViewContextExtensions.cs]: Yes

**Pull request merged**